controllers/tasks.js

```diff
@@ -0,0 +1,101 @@
+const Task = require('../models/Task');
+
+const addTask = (req, res) => {
+    res.render('pages/addTask');
+};
+
+const createTask = async (req, res) => {
+    try {
+      if (req.body.complete) {
+        req.body.completed = true;
+      }
+      await Task.create(req.body);
+      req.session.pendingMessage = "The task was created.";
+      res.redirect("/tasks");
+    } catch (err) {
+      if (err.name === "ValidationError") {
+        res.locals.message = Object.values(err.errors)
+          .map((item) => item.message)
+          .join(", ");
+      } else {
+        res.locals.message = "Something went wrong.";
+      }
+      res.render("pages/addTask");
+    }
+  };
+
+const deleteTask = async (req, res) => {
+    let task = false;
+    try {
+      if (req.body.complete) {
+        req.body.completed = true;
+      }
+      task = await Task.findById(req.params.id);
```

Comment:
Do we need this line?

Reply:
Good day, please check the changes in updated pull request week13 that addressed the changes: deleted unnecessary  "if" function in deleteTask and and deleted the duplicating message.js file.